packages/manager/modules/iplb/src/translations/Messages_fr_FR.json

```diff
@@ -1,3 +1,3 @@
 {
-  "iplb_title": "LoadBalancer"
+  "iplb_title": "OVHcloud Load Balancer"
```

Comment:
**question:** If content is not meant to be translated, couldn't we use it directly as a constant?